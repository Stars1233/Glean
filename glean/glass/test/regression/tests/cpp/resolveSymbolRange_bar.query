action: symbolLocation
args: "test/cpp//foo/bar/.decl"
