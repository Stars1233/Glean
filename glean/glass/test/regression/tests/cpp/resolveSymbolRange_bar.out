[
    "@generated",
    {
        "location": {
            "filepath": "test.cpp",
            "range": {
                "columnBegin": 1,
                "columnEnd": 2,
                "lineBegin": 33,
                "lineEnd": 43
            },
            "repository": "test"
        },
        "revision": "testhash"
    }
]